content/en/serverless/custom_metrics/_index.md

```diff
@@ -3,56 +3,54 @@ title: Custom Metrics
+With log-based metrics, you can record a count of logs that match a query or summarize a numeric value contained in a log, such as request duration. Log-based metrics are a cost-efficient way to summarize log data from the entire ingest stream. Learn more about creating log-based metrics [here][4]. 
 
-**Asynchronous (recommended)**: It's possible to submit your custom metrics with zero latency overhead **and** have them appear in Datadog in near-real-time. To accomplish this, the Lambda Library either:
-- Publishes your custom metrics to the [Datadog Lambda Extension][4], which will submit your custom metrics directly to Datadog without impacting your function latency.
-- Emits your custom metrics as specially-formatted log which the [Datadog Forwarder][5] parses and submits to Datadog. Logging in AWS Lambda is 100% asynchronous, so this method ensures there is zero latency overhead to your function.
+You can also generate metrics from 100% of ingested spans, regardless of whether they are indexed by a retention filter. Learn more about creating span-based metrics [here][5].
+## With the Datadog Lambda Extension
 
-### Enabling asynchronous custom metrics
+{{< img src="serverless/serverless_custom_metrics.png" alt="Collecting Custom Metrics from AWS Lambda" >}}
 
-#### With the Datadog Lambda Extension
+Datadog recommends using the [Datadog Lambda Extension][1] to submit custom metrics from supported Lambda runtimes.
 
-Datadog recommends using the Datadog Lambda Extension if your you are using Node.js or Python Lambda runtimes.
+1. Follow the general [serverless installation instructions][6] to configure your Lambda function and install the Datadog Lambda Library and Extension.
+1. If you are not interested in collecting traces from your Lambda function, set the environment variable `DD_TRACE_ENABLED` to `false`.
+1. If you are not interested in collecting logs from your Lambda function, set the environment variable `DD_SERVERLESS_LOGS_ENABLED` to `false`.
+1. Import and use the helper function from the Datadog Lambda Library, such as `lambda_metric` or `sendDistributionMetric`, to submit your custom metrics following the [sample code](#custom-metrics-sample-code).
 
-1. Set the environment variable `DD_FLUSH_TO_LOG` to `True` on your Lambda function.
-2. Add the [Datadog Lambda Extension][4] to your function either as a Lambda Layer or as a container image.
-3. Add the environment variable `DD_API_KEY` or `DD_KMS_API_KEY` to your Lambda function, and set the value to your Datadog API key on the [API Management page][12]. 
+If your Lambda function is associated with a VPC, ensure that your function can reach Datadog API endpoints either through the public internet, [PrivateLink][7] or a [proxy][8].
```

Comment:
```suggestion
If your Lambda function is running in a VPC, ensure that your function can reach Datadog API endpoints either through the public internet, [PrivateLink][7] or a [proxy][8].
```

Reply:
I've never heard "associated with" a VPC, so just double checking that's what you meant

Reply:
Haha, ask Stephen :)